### ISSUE TITLE
Fix ADA violations

### ISSUE THREAD

**Issue by ghost** [DivanteLtd/next#418]
Fix ADA violations (details: https://www.w3cag.org/ WCAG2-AAA Validator)
- "The html element should have a lang or xml:lang attribute which describes the language of the document."
- "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined."
- "This button element does not have a name available to an accessibility API. Valid names are: title undefined, element content, aria-label undefined, aria-labelledby undefined."
- "Anchor element found with a valid href attribute, but no link content has been supplied."
- "Anchor element found with a valid href attribute, but no link content has been supplied."
